002-vm-boot-from-iso/Makefile

```diff
@@ -0,0 +1,45 @@
+vmname="Haiku"
+iso=haiku-r1alpha4.iso
+remote_iso=/ipfs/QmW9on9UeKCVDb5dsTEZJQwQoH53gXUtVfWvSBhf2Thmyq/isos/haiku-r1alpha4.1-iso.tar.xz
+
+ifeq (, $(shell which VBoxManage))
+  $(error "No VBoxManage found in $(PATH). Please install VirtualBox: https://www.virtualbox.org/wiki/Downloads")
+endif
+
+define notfound
+$(iso) not found
+Make sure the daemon is running and mounted:
+	ipfs daemon &
+	sleep 10
+	ipfs mount
+
+If you are having trouble mounting IPFS, please see https://github.com/jbenet/go-ipfs/blob/master/docs/fuse.md
+endef
+export notfound
```

Comment:
I don't know. I usually don't look at how I'd organize complexity until either it is there or I have experience with it. I've rarely made Makefiles so I don't know yet what I'd want to do when there are a lot of them with snippets like that.